README.md

```diff
@@ -14,14 +13,12 @@ from one endpoint to another service.
   - [data-collector](https://github.com/ManageIQ/aiops-data-collector)
   - [publisher](https://github.com/ManageIQ/aiops-publisher)
 * Discover all AI services we're integrating with
-  - [dummy-ai](https://github.com/ManageIQ/aiops-dummy-ai-service)
-  - [aicoe-insights-clustering](https://github.com/RedHatInsights/aicoe-insights-clustering)
+  - [aiops-dummy-ai-service](https://github.com/ManageIQ/aiops-ai-library/tree/dummy)
```

Comment:
How about excluding this entry from README?

Reply:
I don't really care. Both is possible.. I would say let's keep it in the list for the visibility, though it serves no purpose, so it can be removed as well.. Nah. :smile:

Reply:
Ok, lets keep it :)